ci/prow-build-test-qemu.sh

```diff
@@ -30,7 +30,9 @@ fi
 ocpver=$(rpm-ostree compose tree --print-only src/config/manifest.yaml | jq -r '.["mutate-os-release"]')
 ocpver_mut=$(rpm-ostree compose tree --print-only src/config/manifest.yaml | jq -r '.["mutate-os-release"]' | sed 's|\.|-|')
 prev_build_url=${REDIRECTOR_URL}/rhcos-${ocpver}/
-curl -L http://base-"${ocpver_mut}"-rhel8.ocp.svc.cluster.local > src/config/ocp.repo
+# we want to use RHEL 8.5 for testing until we can start using 8.6
+# see https://github.com/openshift/release/pull/26193
+curl -L http://base-"${ocpver_mut}"-rhel85.ocp.svc.cluster.local > src/config/ocp.repo
```

Comment:
This is fine as is, but it's another place to bump the RHEL version.  Let's figure out how to dedup all this after recent PRs merge though, don't block here!

Reply:
Ack.  Will do!